frontend/src/components/pages/SearchBox.tsx

```diff
@@ -0,0 +1,45 @@
+import { Search2Icon } from "@chakra-ui/icons";
+import {
+  Button,
+  Input,
+  InputGroup,
+  InputLeftElement,
+  InputRightElement,
+} from "@chakra-ui/react";
+import React from "react";
+
+const SearchBox = (): React.ReactElement => {
```

Comment:
Hey, so this ticket also involves keeping track of the search text as state. I think the previous dev forgot to implement this. Could you:
1. Take a function called `setSearchText` as a prop
2. Upon click of the search button, call this function with the entered search text

Thanks!